src/main/java/io/kestra/plugin/amqp/Consume.java

```diff
@@ -114,10 +136,10 @@ private boolean ended(AtomicInteger count, ZonedDateTime start, Integer max, Dur
         if (max != null && count.get() >= max) {
             return true;
         }
-        if (duration != null && ZonedDateTime.now().toEpochSecond() > start.plus(duration).toEpochSecond()) {
+        if (duration != null
+            && ZonedDateTime.now().toEpochSecond() > start.plus(duration).toEpochSecond()) {
```

Comment:
Same and the rest of files... Everything except the  `@Metric` should be removed